includes/cf/sonarr-hq-source-group.md

```diff
@@ -1,10 +1,14 @@
 ??? summary "HQ Source Groups - [CLICK TO EXPAND]"
-    | Custom Format                                                                                                   | Score                                            | Trash ID                                      |
-    | --------------------------------------------------------------------------------------------------------------- | ------------------------------------------------ | --------------------------------------------- |
-    | [{{ sonarr['cf']['web-tier-01']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#web-tier-01)           | {{ sonarr['cf']['web-tier-01']['trash_score'] }} | {{ sonarr['cf']['web-tier-01']['trash_id'] }} |
-    | [{{ sonarr['cf']['web-tier-02']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#web-tier-02)           | {{ sonarr['cf']['web-tier-02']['trash_score'] }} | {{ sonarr['cf']['web-tier-02']['trash_id'] }} |
-    | [{{ sonarr['cf']['web-tier-03']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#web-tier-03)           | {{ sonarr['cf']['web-tier-03']['trash_score'] }} | {{ sonarr['cf']['web-tier-03']['trash_id'] }} |
-    | [{{ sonarr['cf']['web-scene']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#web-scene) :exclamation: | {{ sonarr['cf']['web-scene']['trash_score'] }}   | {{ sonarr['cf']['web-scene']['trash_id'] }}   |
+    | Custom Format                                                                                                     | Score                                                  | Trash ID                                            |
+    | ----------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ | --------------------------------------------------- |
+    | [{{ sonarr['cf']['remux-tier-01']['name'] }}](/sonarr/sonarr-collection-of-custom-formats/#remux-tier-01)         | {{ sonarr['cf']['remux-tier-01']['trash_score'] }}     | {{ sonarr['cf']['remux-tier-01']['trash_id'] }}     |
+    | [{{ sonarr['cf']['remux-tier-02']['name'] }}](/sonarr/sonarr-collection-of-custom-formats/#remux-tier-02)         | {{ sonarr['cf']['remux-tier-02']['trash_score'] }}     | {{ sonarr['cf']['remux-tier-02']['trash_id'] }}     |
```

Comment:
This shouldn't be added here, because then the remux tier will also be shown in the webdl guide section for Sonarr